sources/parse-datetime/fuzz/fuzz_targets/parse.rs

```diff
@@ -0,0 +1,6 @@
+#![no_main]
```

Comment:
Is this part of ghostdog? If not, it should go in another repo.

Reply:
this is not a vendored library as far as I can tell. "parse-datetime" is not on crates.io, and it seems like a small utility library for them. which is pretty common in rust, to break out standalone things into crates (it helps organization and compile times)